specs/fast-project-realignment/README.md

```diff
@@ -0,0 +1,210 @@
+
+#### State of next packages
+- [Removed FoundationElement and the dependency injection system](https://github.com/microsoft/fast/pull/6160) (i.e. `tagFor`).
+- Breaking [public API template changes](https://github.com/microsoft/fast/pull/6286) in FAST Foundation.
+
+#### Users of next packages
+- [Adaptive Web Components](https://github.com/Adaptive-Web-Community/Adaptive-Web-Components/tree/%40adaptive-web/adaptive-web-components_v0.8.1/packages/adaptive-web-components)
+
+### Stable packages
+
+See [FAST](https://github.com/microsoft/fast) and relevant package: [`@microsoft/fast-element`](https://github.com/microsoft/fast/tree/master/packages/web-components/fast-element)
+
+#### Users of stable packages
+- [`@fluentui/web-components`](https://github.com/microsoft/fluentui/tree/master/packages/web-components) v3 (in beta)
+    - Have inlined the FAST Foundation [templates/classes](https://github.com/microsoft/fluentui/pull/30090) and [utilities](https://github.com/microsoft/fluentui/pull/31009) in a non-shareable way.
+    - [Removed Design Token infrastructure](https://github.com/microsoft/fluentui/pull/30002).
+
+## Links To Relevant Work Items and Reference Material
+
+HLD Work item: [#2207](https://github.com/ni/nimble/issues/2207)
+
+## Implementation / Design
+
+### Priorities
+
+- Unblock Nimble bug fixes and new feature development in the short-term (scope of this HLD).
+- Align on a path for long-term maintainability of Nimble (Future work).
+
+### Minimal fork proposal
+
+Work that should be done imminently (over the next 1-3 months).
+
+ - Fork the `microsoft/fast` repo to `ni/fast` as a new top-level GitHub repository.
+- Make the `archives/fast-element-1` branch the default branch:
+    - Keep the name `archives/fast-element-1` on the default branch.
+    - Delete other branches.
+- Update the packages to use the `ni` scope:
+    - `@microsoft/fast-colors` to `@ni/fast-colors`.
+    - `@microsoft/fast-element` to `@ni/fast-element`.
+    - `@microsoft/fast-foundation` to `@ni/fast-foundation`.
+    - `@microsoft/fast-web-utilities` to `@ni/fast-web-utilities`.
+    - `@microsoft/fast-react-wrapper` to `@ni/fast-react-wrapper`.
+- Remove unused packages and code (refer to [`archives/fast-element-1`](https://github.com/microsoft/fast/tree/archives/fast-element-1)):
+   
+   ```
+   /examples
+   /packages/tooling
+   /packages/utilities/fast-benchmarks
+   /packages/utilities/fast-eslint-rules
+   /packages/web-components/fast-router
+   /sites
+   /specs
+   ```
+
+- Update npm dependencies to latest.
```

Comment:
Some other common repo things we should probably do:
1. enable Renovate
1. add CODEOWNERS (can happen after all the setup is done, but we want changes to be reviewed)
1. update README to link back to the original repo and graciously acknowledge that we are standing on their shoulders
1. ensure we are in compliance with their license and pick what license we will use for the fork
2. update CONTRIBUTING to describe policies for beachball / owners (could be copied from Nimble or just link to Nimble's)